lib/patterns.coffee

```diff
@@ -83,11 +84,20 @@ exports.login = (callback) ->
 #		console.log(token)
 ###
 exports.loginWithToken = (callback) ->
-	form.ask
-		label: 'What\'s your token? (visible in the preferences page)'
-		name: 'token'
-		type: 'text'
-	, callback
+	###
+	Use of prompt package instead of inquirer in order to
+	address bug declared in https://github.com/resin-io/resin-cli/issues/11, which results in
+	repeated output of token when it is given as an input to the command-line through copy & paste.
+	###
+	prompt.message = '?'.green
+	prompt.delimiter = ' '
+	prompt.start()
+	prompt.get [
+		name: 'token',
```

Comment:
The comma is not necessary in CoffeeScript, so better to remove it for consistency.